cartesi-rollups/development/running-the-application.md

```diff
@@ -17,7 +17,7 @@ NoNodo is a light local Cartesi node for development and testing that does not r
 
```

Comment:
I don't like saying nonodo is a local cartesi node. It lacks essential things from cartesi, we should call it something else. A tool that spins up a local network/blockchain and you can run your dapp comunicating with it without the cartesi machine